### DISCUSSION TITLE
版面分析结果，有的切割不对，怎么改善？

### DISCUSSION THREAD

**Discussion by nissansz** [RapidAI/RapidOrientation#7] [Q&A]
请提供下述完整信息以便快速定位问题
(Please provide the following information to quickly locate the problem)
- **系统环境/System Environment**：
- **使用的是哪门语言的程序/Which programing language**：
- **所使用语言相关版本信息/Version**：
- **OnnxRuntime版本/OnnxRuntime Version**：
- **可复现问题的demo/Demo of reproducible problems**：
- **完整报错/Complete Error Message**：
- **可能的解决方案/Possible solutions**:

切割错误举例如下
![image](https://github.com/RapidAI/RapidStructure/assets/41010669/d3d21890-3fd8-490e-8c50-9f880fbf6e8b)
{'bbox': array([  10.66207372, 2231.28837461, 1073.79497789, 2344.79886966]), 'label': 'text'}
{'bbox': array([ 500.75091953,   12.21875445, 1613.09947578,   70.05863229]), 'label': 'text'}
{'bbox': array([   7.98665632, 2364.51628573, 2081.99992877, 2901.74100257]), 'label': 'figure'}
{'bbox': array([2.87512385e-01, 7.39058257e+02, 2.08199993e+03, 2.22017507e+03]), 'label': 'table'}
{'bbox': array([   4.60234904,  334.93956088, 2062.68867985,  693.52744136]), 'label': 'table'}
{'bbox': array([2.03777382e+00, 8.42523906e+01, 2.05676338e+03, 4.27240529e+02]), 'label': 'table'}
{'bbox': array([ 84.74693863, 712.07451807, 846.26481067, 750.32085518]), 'label': 'table_caption'}

原图如下
![dianfei](https://github.com/RapidAI/RapidStructure/assets/41010669/68bc60df-eb33-4eac-8178-9e47969dddef)

**Comment by tslxxcxy** [comment]
你这个是通过版面分析，然后再识别的吗
